projects/fsastorefrontlib/src/core/my-account/facade/claim.service.ts

```diff
@@ -28,21 +28,21 @@ export class ClaimService {
       .subscribe(([claim, userLoggedIn]) => {
         this.currentClaimId = claim.claimNumber;
         if (this.isCreated(claim) && userLoggedIn) {
-          this.loadCurrentClaim();
+          this.loadClaimById(this.currentClaimId);
         }
       })
       .unsubscribe();
   }
 
-  loadCurrentClaim() {
+  loadClaimById(claimId) {
```

Comment:
We should think about deprecating methods and all action names which you changed, since we already had release or we should create some page where we would explain in details what needs to be replaced.